x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -233,6 +233,11 @@ export const allowedExperimentalValues = Object.freeze({
    * Makes Elastic Defend integration's Malware On-Write Scan option available to edit.
    */
   malwareOnWriteScanOptionAvailable: false,
+
+  /**
+   * Enables the new modal for the value list items
+   */
+  valueListItemsModal: true,
```

Comment:
consider renaming this to `valueListItemsModalEnabled` as the most of the flags above :)